workflows/yelp.com/workflows.arazzo.json

```diff
@@ -416,6 +416,186 @@
           "reference": "$components.failureActions.server_error"
         }
       ]
+    },
+    {
```

Comment:
We have to delete this workflow. Unless we have this https://github.com/OAI/Arazzo-Specification/issues/366, we wouldn't be able to run this workflow.